docs/development.md

```diff
@@ -128,7 +128,15 @@ and the local development version of msprime.
 (sec_development_documentation_building)=
 ### Building
 
-To build the documentation locally, go to the `docs` directory and run `make`.
+To build the documentation locally, follow the 
+{ref}`Quickstart <sec_development_quickstart>`. Then install the python
+requirements for building the documentation:
+
+```bash
+pip install -r requirements/CI-docs/requirements.txt
```

Comment:
Hmm, not sure we want to do this. The CI requirements are all pinned, which will cause issues for some people.

We should just add the packages needed to the ``requirements/development.txt`` - which ones were missing?

Reply:
matplotlib was the one I saw. There might be more. I'll debug and find out.